docs/troubleshooting.rst

```diff
@@ -83,3 +83,6 @@ This script also supports specifying a cookie file path, which may be useful if
 you use multiple Firefox profiles or if your operating system has the directory
 structure differently set up. Also, you can specify an alternative session file
 path.
+
+If this script doesn't work for you (for example because Firefox locks the sqlite file),
```

Comment:
Two ideas here:

- With connecting to the file with immutable=1 (which still has to be changed), Firefox locking the file is not a reason for 615_import_firefox_session.py to not work, so we should choose another example, or remove the "for example" clause.
- Maybe (not so sure about that, just want to suggest it), "If this script doesn't work for you" should be written in bold (surrounded by two asterisks `**`) to be more visible when being frustrated and not reading all the text.